JSQMessagesViewController/Controllers/JSQMessagesViewController.m

```diff
@@ -903,8 +904,12 @@ - (void)jsq_adjustInputToolbarHeightConstraintByDelta:(CGFloat)dy
     
     if (self.toolbarHeightConstraint.constant < kJSQMessagesInputToolbarHeightDefault) {
         self.toolbarHeightConstraint.constant = kJSQMessagesInputToolbarHeightDefault;
+    } else if (_maxToolbarHeight > 0) {
```

Comment:
I also guess `maximumInputToolBarHeight != NSNotFound` should be evaluated now instead of `> 0`